beps/0005-split-backend-discovery/README.md

```diff
@@ -0,0 +1,167 @@
+---
+title: Discovery API - Split Backends
+status: provisional
+authors:
+  - '@aramissennyeydd'
+owners:
+project-areas:
+  - core
+creation-date: 2024-02-16
+---
+
+<!--
+**Note:** When your BEP is complete, all these pre-existing comments should be removed
+
+When editing BEPs, aim for tightly-scoped, single-topic PRs to keep discussions focused. If you disagree with what is already in a document, open a new PR with suggested changes.
+-->
+
+# BEP: Discovery API - Split Backends
+
+<!-- Before merging the initial BEP PR, create a feature issue and update the below link. You can wait with this step until the BEP is ready to be merged. -->
+
+[**Discussion Issue**](https://github.com/backstage/backstage/issues/NNNNN)
+
+- [Summary](#summary)
+- [Motivation](#motivation)
+  - [Goals](#goals)
+  - [Non-Goals](#non-goals)
+- [Proposal](#proposal)
+- [Design Details](#design-details)
+- [Release Plan](#release-plan)
+- [Dependencies](#dependencies)
+- [Alternatives](#alternatives)
+
+## Glossary
```

Comment:
I'm worried this glossary is a little bit too prescriptive to lead the entire BEP, probably better to move this down to the design section instead.